tests/installation/clock_and_timezone/validate_timezone_configuration.pm

```diff
@@ -7,24 +7,24 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Handles Time and Date dialog in YaST Firstboot Configuration.
-#
+# Summary: Collects all the info about current Timezone configuration
+# and validates it against the one provided by test_data.
 # Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
 
-use base 'y2_firstboot_basetest';
+use base 'y2_installbase';
 use strict;
 use warnings;
 use scheduler 'get_test_suite_data';
 use cfg_files_utils;
 
 sub run {
-    my $test_data           = get_test_suite_data()->{time_and_date};
+    my $test_data = get_test_suite_data()->{time_and_date};
+
     my $clock_and_time_zone = $testapi::distri->get_clock_and_time_zone();
     compare_settings({
             expected      => $test_data,
             current       => $clock_and_time_zone->collect_current_clock_and_time_zone_info(),
             suppress_info => 1});
-    $clock_and_time_zone->proceed_with_current_configuration();
```

Comment:
could you please delete after this change [proceed_with_current_configuration](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/dbd780e4b3df2278e28bc9e113803864382c3d8f/lib/Installation/ClockAndTimeZone/ClockAndTimeZoneController.pm#L47) [btn_next](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/dbd780e4b3df2278e28bc9e113803864382c3d8f/lib/Installation/ClockAndTimeZone/ClockAndTimeZonePage.pm#L27) and [press_next](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/dbd780e4b3df2278e28bc9e113803864382c3d8f/lib/Installation/ClockAndTimeZone/ClockAndTimeZonePage.pm#L54)

Reply:
Done.